device/accton/x86_64-accton_csp7551-r0/Accton-CSP7551/switch-sai.conf

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This file is not needed for P4_16 SDK any more.

Reply:
Ok, I'll delete it.